smf/internal-dns/config.toml

```diff
@@ -0,0 +1,22 @@
+[dropshot]
+bind_address = "[fd00:1234::4]:5353"
+request_body_max_bytes = 1048576
+
+[dns]
+bind_address = "[fd00:1234::4]:4753"
+
+[log]
+# Show log messages of this level and more severe
+level = "debug"
+
+# Example output to a terminal (with colors)
+mode = "stderr-terminal"
+
+# Example output to a file, appending if it already exists.
+#mode = "file"
+#path = "logs/server.log"
+#if_exists = "append"
+
+[data]
+nmax_messages = 16
+storage_path = "/var/tmp/oxide/dns"
```

Comment:
`/var/dns`?  I'd defer to @jclulow on this but `/var/tmp` doesn't seem right.

Reply:
This is where the sled kv store data goes. So if we want that to be persistent across reboots `/var/dns` would be better.

Reply:
Updated